ci/pipeline.yaml

```diff
@@ -233,3 +248,64 @@ jobs:
                 ! diff --context=3 \
                   <(tar xzfO nginx-chart/*.tgz nginx/values.yaml) \
                   <(tar xzfO rewritten-chart/chart.tgz nginx/values.yaml)
+
+
+  - name: test-pull-request
+    serial: true
+    plan:
+      - in_parallel:
+        - get: test-image
+          passed:
+            - build-test-image
+        - get: source
+          resource: pull-requests
+          trigger: true
+          version: every
+          params:
+            integration_tool: rebase
+      - put: pull-requests
+        inputs: detect
```

Comment:
probably basic question but what does `        inputs: detect` do?

Reply:
It's a concourse thing, I believe by default, every resource pulled and `output` of a task will be mounted as a file system in the container for the step being run. However, you don't often need every resource. Using `inputs: detect` will attempt to reduce the number of volume mounts to only what it needs. This means smaller running containers, meaning faster and less resource intensive.

I think this will eventually become the default.

Reply:
Interesting, thanks for the explanation!